.travis.yml

```diff
@@ -52,13 +52,13 @@ matrix:
     bundler_args: --without changelog development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
   - env:
       CHEFSTYLE: 1
-    rvm: 2.1
+    rvm: 2.2.5
```

Comment:
Curious, if we plan to move to 2.3.1, why not set these to that instead of 2.2.5?